behave_test_automation/pah_behave/features/steps/steps.py

```diff
@@ -0,0 +1,10 @@
+import time
+from behave import given, then, when
+import features.steps.selectors as selector
+
+
+@when(u'User enters "{text}" into "{label}" field')
+def step_impl(context, text, label):
+    input_element = context.driver.find_element_by_xpath("//label[text()='" + label + "']/../input")
```

Comment:
Maybe could you use f-string?

Reply:
Did not refactor this step yet, since it was not a part of `login-testcase.feature`. Will do it in next PR since I want to provide smaller ones :)